.pre-commit-config.yaml

```diff
@@ -26,3 +26,5 @@ repos:
       hooks:
           - id: black
             language_version: python3.10
+            args:
+                - "--line-length=120"
```

Comment:
Now we have it duplicated, the same is in .flake8, probably needs to be adressed in the linting ADR/follow up stories. Feel free to resolve.

Reply:
As I understand, flake8 is not performing linting changes, just checks. And black is set to a default line length of 88. So this is not a duplicate, but passing the appropriate config to black.